atomic_reactor/plugins/post_export_operator_manifests.py

```diff
@@ -0,0 +1,138 @@
+"""
+Copyright (c) 2019 Red Hat, Inc
+All rights reserved.
+
+This software may be modified and distributed under the terms
+of the BSD license. See the LICENSE file for details.
+"""
+
+import os
+import shutil
+import tarfile
+import tempfile
+
+from atomic_reactor.constants import (PLUGIN_EXPORT_OPERATOR_MANIFESTS_KEY,
+                                      OPERATOR_MANIFESTS_ARCHIVE)
+from atomic_reactor.plugin import PostBuildPlugin
+from atomic_reactor.util import df_parser, is_scratch_build, get_platforms
+from docker.errors import APIError
+from osbs.utils import Labels
+
+MANIFESTS_DIR_NAME = 'manifests'
+OPERATORS_ARCHIVE_NAME, OPERATORS_ARCHIVE_EXT = OPERATOR_MANIFESTS_ARCHIVE.rsplit('.', 1)
+
+
+class ExportOperatorManifestsPlugin(PostBuildPlugin):
+    """
+    Export operator manifest files
+
+    Fetch and archive operator manifest files from image so they can be
+    uploaded to koji.
+    """
+
+    key = PLUGIN_EXPORT_OPERATOR_MANIFESTS_KEY
+    is_allowed_to_fail = False
+
+    def __init__(self, tasker, workflow, operator_uploader_platform=None, platform=None):
+        super(ExportOperatorManifestsPlugin, self).__init__(tasker, workflow)
+
+        self.operator_uploader_platform = operator_uploader_platform
+        if platform:
+            self.platform = platform
+        else:
+            self.platform = os.uname()[4]
```

Comment:
Isn't this equivalent to https://docs.python.org/2/library/platform.html#platform.machine

Reply:
It is. I did not know about platform.machine :)

Note that there are other parts of the code using os.uname to fetch the arch name. Maybe we should also change those for consistency if we are going with platform.machine here.

Reply:
I'd avoid changing it everywhere for now